### ISSUE TITLE
关于模型两部分输出的问题

### ISSUE THREAD

**Issue by xiaochus** [cardwing/Codes-for-Lane-Detection#47]
你好，我看到tf实现的模型中，lane detection的输出是卷积结果直接resize后输出，而softmax被放入了lane existence prediction branch，如下：
```
           ret['prob_output'] = tf.image.resize_images(conv_output, [CFG.TRAIN.IMG_HEIGHT, 
           CFG.TRAIN.IMG_WIDTH])

            ### add lane existence prediction branch ###

            # spatial softmax #
            features = conv_output  # N x H x W x C
            softmax = tf.nn.softmax(features)

            avg_pool = self.avgpooling(softmax, kernel_size=2, stride=2)
```
这个实现与论文中Fig 5给出的结构有些不同，请问这个改动有什么作用呢？

**Comment by cardwing** [comment]
@xiaochus，tf的模型是按照torch版本的SCNN改过来的，也就是说原版的SCNN的softmax就只作用在了lane existence prediction branch。同时，这里的softmax是spatial softmax，也就是让输出的feature在spatial（height X width）维度上做一个归一化。下面这个文件就是torch版本的SCNN模型，你可以看一下，我推断论文里面的图可能画错了。

**Comment by xiaochus** [comment]
@cardwing thanks!
